README.md

```diff
@@ -15,6 +21,16 @@ Included are the following:
 - testscript: script-based testing based on txtar files
 - txtar: simple text-based file archives for testing.
 
-# Links
+### testscript
```

Comment:
Did you intend for this to be a level 4 heading below `### packages`?

Reply:
(A level 4 heading made testscript maybe too small, so went the other direction and made Packages a level 2 heading).